config/config.go

```diff
@@ -32,6 +37,14 @@ type Application struct {
 	TransactionActiveLimit     int      `yaml:"transaction_active_limit"`
 }
 
+// Log is used to manage the SPOA logging.
+type Log struct {
+	Level     string `yaml:"level"`
```

Comment:
I don't think we should provide two ways to do login in a connector. Can we stick to tx's logger?

Reply:
No, we need application (`coraza-spoa`) logging. We need to log all initialization stuff as well as request/response before transaction is created.